bindings/python/setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools_rust import Binding, RustExtension
 
 extras = {}
-extras["testing"] = ["pytest"]
+extras["testing"] = ["pytest", "requests", "numpy", "datasets"]
```

Comment:
I propose to add these dependencies that I had to install to run the python tests.

Reply:
Fine for me.